packages/newspringchurchapp/src/ui/PrayerMenu/__snapshots__/PrayerMenu.tests.js.snap

```diff
@@ -0,0 +1,1034 @@
+        snapToAlignment="start"
+        snapToInterval={512}
+        stickyHeaderIndices={Array []}
+        style={
+          Object {
+            "paddingLeft": 16,
+          }
+        }
+        updateCellsBatchingPeriod={50}
+        viewabilityConfigCallbackPairs={Array []}
+        windowSize={21}
+      >
+        <View>
+          <View
+            onLayout={[Function]}
+            style={
+              Array [
+                Object {
+                  "flexDirection": "row",
+                },
+                null,
+              ]
+            }
+          >
+            <View
+              accessible={true}
+              onResponderGrant={[Function]}
+              onResponderMove={[Function]}
+              onResponderRelease={[Function]}
+              onResponderTerminate={[Function]}
+              onResponderTerminationRequest={[Function]}
+              onStartShouldSetResponder={[Function]}
+              style={
+                Object {
+                  "transform": Array [
+                    Object {
+                      "scale": 1,
+                    },
+                  ],
+                }
+              }
+            >
+              <View
+                style={
+                  Object {
+                    "aspectRatio": 1,
+                    "height": 120,
+                    "marginRight": 12,
+                  }
+                }
+              >
+                <View
+                  style={
+                    Object {
+                      "aspectRatio": 1,
+                      "borderRadius": 16,
+                      "height": "100%",
+                      "overflow": "hidden",
+                      "width": "100%",
+                    }
+                  }
+                >
+                  <View
+                    style={
+                      Object {
+                        "aspectRatio": 1,
+                        "backgroundColor": "#dddddd",
+                        "overflow": "hidden",
+                        "width": "100%",
+                      }
+                    }
+                  >
+                    <View
+                      animate={null}
+                      customAnimate={null}
+                      onReady={false}
+                      style={
+                        Object {
+                          "aspectRatio": 1,
+                          "backgroundColor": "#dddddd",
+                          "width": "100%",
+                        }
+                      }
+                    >
+                      <Image
+                        onLoad={[Function]}
+                        source={
+                          Array [
+                            Object {
+                              "cache": "force-cache",
+                              "uri": "https://picsum.photos/600/400/?random",
+                              "url": "https://picsum.photos/600/400/?random",
```

Comment:
A url and an uri?

Reply:
I see this in TAP snapshots so I'm guessing it's okay.